salesforce/backend/query.py

```diff
@@ -483,6 +513,12 @@ def execute(self, q, args=()):
             self.results = iter([])
 
     def execute_select(self, q, args):
+        if q == "SELECT django_migrations.app, django_migrations.name FROM django_migrations":
```

Comment:
I never would have anticipated fixes like this. Do you have any documentation that I don't know? In general I can't find any docs on how to write a custom dajngo db backend, which is why I guess you have to live with lots of undocumented and "tricky" code.